### ISSUE TITLE
Is there a graph in HPRC that contains HG002？Or is there a vcf file that is unique to HG002?

### ISSUE THREAD

**Issue by tanger-code** [ComparativeGenomicsToolkit/cactus#1191]
Hi!
Is there a graph in HPRC that contains HG002？Or is there a vcf file that is unique to HG002?
I took a look at the file directory for [HPRC ](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=pangenomes/freeze/freeze1/minigraph-cactus/) and it doesn't seem to have this information.

**Comment by glennhickey** [comment]
No, there isn't.  This has annoyed a fair number of people.  Next release we'll make a graph with all samples (and use a separate held-out graph for evaluation).  For now, if you want HG002 you're stuck making your own graph.
